inventory/group_vars/all.yml

```diff
@@ -191,11 +191,13 @@ dns_server: "{{ kube_service_addresses|ipaddr('net')|ipaddr(2)|ipaddr('address')
 # Path used to store Docker data
 docker_daemon_graph: "/var/lib/docker"
 
+docker_iptables: false
```

Comment:
That's not how all.yml file works. We don't stuff every possible option here. We only put the ones users are likely to want to change. Please move it to roles/docker/defaults/main.yml and default to false. It actually breaks deployments when set to true, so I doubt we want to invite users to re-enable it.